plugins/database/parameters.go

```diff
@@ -13,8 +14,16 @@ const (
 	CfgDatabaseDirty = "database.dirty"
 )
 
+// Parameters contains configuration parameters used by the storage layer.
+var Parameters = struct {
+	// ForceCacheTime  is a new global cache time in seconds for object storage.
+	ForceCacheTime int `default:"-1" usage:"number of seconds all objects should remain in memory. -1 means use defaults"`
```

Comment:
We can use `time.Duration` Instead of an int, the configuration framework will parse it for us.

It's worth updating the description then: 
`default:"-1s" usage:"interval of time objects should remain in memory. Zero time means no caching, negative value means use defaults"`